assets/css/style.css

```diff
@@ -196,13 +225,13 @@ h4:hover a {
 }
 
 .section {
-    padding-top: 80px;
+    padding-top: calc(210px - 4vw);
```

Comment:
Is this duplicated in the HTML layout? See here:

https://github.com/unicef/inventory-hugo-theme/pull/10/commits/1c3a979f7051db590ae52ee6227854e965fffbc9#diff-3edcca306992c0cab9f171119e108a629ef56d8e5310654a54d59d6a2791cc6aR2

Reply:
@jwflory not really.  Let me get rid off the repitition though.